### PR TITLE
Adding support for Ubuntu 16.04 via systemd service support

### PR DESCRIPTION
**Pull request by cdoughty-r7**
Currently this cookbook does not support systems that depend on systemd to start services.  

This change adds support for systemd.

### CODE REVIEW

**Comment by cdoughty-r7** [comment]
@fmitchell-r7 @dgreene-r7 Ready for review, if either of you get a chance.

File: Gemfile [hunk 1]
```diff
@@ -8,4 +8,8 @@ gem 'mime-types', '~> 3.1'
 
 group :cookbook do
   gem 'builderator', '~> 1.0'
+  gem 'test-kitchen'
+  gem 'kitchen-docker'
```
Comment (erran-r7) [inline Gemfile]: Why include the docker driver if we don't use it?

Comment (cdoughty-r7) [inline Gemfile]: copy/paste.  I'll remove.

File: Gemfile.lock [hunk 2]
```diff
@@ -136,6 +137,10 @@ GEM
     ipaddress (0.8.3)
     jmespath (1.3.1)
     json (2.0.2)
+    kitchen-docker (2.6.0)
```
Comment (fmitchell-r7) [inline Gemfile.lock]: @cdoughty-r7, can you regenerate the Gemfile.lock so `kitchen-docker` is removed from it as well?

**Comment by cdoughty-r7** [comment]
@fmitchell-r7 Added, thanks for catching that.

**Pull request merged**